IBPSA/Media/Examples/SteamSaturatedProperties.mo

```diff
@@ -0,0 +1,88 @@
+within IBPSA.Media.Examples;
+model SteamSaturatedProperties
+  "Model that tests the implementation of the steam properties at saturated liquid and vapor states"
+  extends Modelica.Icons.Example;
+
+  package Medium = IBPSA.Media.Steam "Steam medium model";
+
+  parameter Modelica.SIunits.Temperature TMin = 273.16
+    "Minimum temperature for the simulation";
+  parameter Modelica.SIunits.Temperature TMax = 647.096
+    "Maximum temperature for the simulation";
+  parameter Modelica.SIunits.Pressure pMin = 620
+    "Minimum pressure for the simulation";
+  parameter Modelica.SIunits.Pressure pMax = 20000000
+    "Maximum pressure for the simulation";
+  Modelica.SIunits.Pressure pSat "Saturation pressure";
+  Medium.Temperature TSat  "Saturation temperature";
+  Modelica.SIunits.Conversions.NonSIunits.Temperature_degC TSat_degC
+    "Celsius saturation temperature";
```

Comment:
I don't see a need to introduce `TSat_degC`. Better is to change the line above to
```
Medium.Temperature TSat(displayUnit="degC")  "Saturation temperature";
```